### PR TITLE
EpiLogo v4.0.0

### PR DESCRIPTION
**Pull request by JakeSaunders1995**
The fourth update to EpiLogo.

- Improved pathfinding using A* 
- Better population initialisation from input files 
- Improve disease mechanics 
- New agent scheduling format

### CODE REVIEW

File: libraries.conf [hunk 1]
```diff
@@ -108,8 +108,8 @@ It allows you to perform unsupervised density-based clustering of turtles and pa
     name: "EpiLogo"
     codeName: "epilogo"
     shortDescription: "An epidemiological package for NetLogo"
-    longDescription: "An epidemiological package for NetLogo containing features such as demographic population initialization, agent pathfinding, agent scheduling and disease campaign initialization/progression. This package allows for easier and less complex development of epidemiological models within NetLogo"
-    version: "3.0.0"
+    longDescription: "An epidemiological package for NetLogo containing features such as demographic population initialization, agent pathfinding, agent scheduling and disease campaigning. This package allows for easy and less complex development of epidemiological models within NetLogo"
+    version: "4.0.0"
     homepage: "https://github.com/JakeSaunders1995/EpiLogo"
     downloadURL: "https://raw.githubusercontent.com/NetLogo/NetLogo-Libraries/6.1/extensions/epilogo-3.0.0.zip"
```
Comment (LaCuneta) [inline libraries.conf]: The `downloadURL` will need to be updated to point to the 4.0.0 zip file.

**Review by LaCuneta — COMMENTED** [review]
Hey Jake,

Thank you for your contribution!  Just a couple small issues.  The zip file name needs to be updated for the `downloadURL` field in the `libraries.conf` file.  Also, the zip file contains an `epilogo` folder instead of directly containing the jar and other files.  There is more explanation in this section of the wiki:  https://github.com/NetLogo/NetLogo-Libraries/wiki/Submission-workflow#extension-submission-or-update

Once those two issues are fixed, we can get this merged.  Let me know if you have any questions.

File: libraries.conf (hunk 1 above)
Comment (JakeSaunders1995) [reply libraries.conf]: Hi this should now be resolved

**Review by LaCuneta — CHANGES_REQUESTED** [review]
This is close, thank you for the updates!  The `epilogo-4.0.0.zip` file is now correctly setup and the `libraries.conf` file is correct, too.

With the last commit, though, an extra folder was added, `extensions/epilogo-4.0.0/` with a couple extra files in it: https://github.com/NetLogo/NetLogo-Libraries/pull/40/commits/3a704a8e5a9a1516bd57e466a13e08357da5026b  That was probably some work you did to create the proper zip file.  Once that folder and files are removed, this will be good to merge.

Thanks again for your contribution.

**Review by LaCuneta — APPROVED** [review]
Looks good, thank you!

**Pull request merged**